sys-devel/qconf/qconf-2.5.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils
+
+DESCRIPTION="./configure like generator for qmake-based projects"
+HOMEPAGE="https://github.com/psi-im/qconf"
+SRC_URI="http://psi-im.org/files/qconf/${P}.tar.xz"
+SRC_URI="https://github.com/psi-im/qconf/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
This is odd. There can only be one `SRC_URI`.

Reply:
Thanks. forgot this thing while tested. fixed now.